### PR TITLE
Fix text action set default result not strictly speaking working

### PR DESCRIPTION
**Pull request by hedvigoscar**
### Checklist

- [X] Functionality is covered by an integration test
- [X] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt [hunk 1]
```diff
@@ -8,9 +8,28 @@ class TextActionSetViewModel : ViewModel() {
     private val _isValid = MutableLiveData<HashMap<Int, Boolean>>(hashMapOf())
     val isValid: LiveData<HashMap<Int, Boolean>> = _isValid
 
+    private val _inputs = MutableLiveData<Map<Int, String>>()
+    val inputs: LiveData<Map<Int, String>> = _inputs
+
```
Comment (hugokallstrom) [inline app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt:13]: Would it be possible to pass TextActionSetData as constructor parameter to the view model? I know it was difficult with dagger :(

**Review by hugokallstrom — APPROVED** [review]

File: app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt:13]: Kind of easy actually: https://insert-koin.io/docs/reference/koin-android/viewmodel#viewmodel-and-injection-parameters

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt:13]: Also, I think that use-case is made easier through Hilt 😉

Comment (hugokallstrom) [reply app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt:13]: Cool! Should be able to pass `parameters: ParametersDefinition` to `Fragment.viewModel()` and then specify the type in the koin module?

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt:13]: Yes, there's good potential here, let me investigate 🙂

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt]: It's glorious!

**Pull request merged**